src/components/Person.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ people }) => {
```

Comment:
you can use destructurization here

Reply:
This component should accept one person, not all people